libs/driver/guides/magento.md

```diff
@@ -0,0 +1,66 @@
+
+# Magento/MageOS
+
+The `@daffodil/driver/magento` package configures the basic integration between your storefront and Magento 2/MageOS ecommerce platforms. 
```

Comment:
The `@daffodil/driver/magento` package configures the integration between your storefront and Magento 2/MageOS ecommerce platforms.